sdk/core/src/request.rs

```diff
@@ -0,0 +1,60 @@
+use http::{HeaderMap, Method};
```

Comment:
I'd like to eventually get rid of any visible dependency on `http`. It would be nice to move in that direction.